subsys/bluetooth/services/dis.c

```diff
@@ -45,105 +48,262 @@ static struct dis_pnp dis_pnp_id = {
 };
 #endif
 
+#if defined(CONFIG_BT_DIS_SYSTEM_ID)
+/*
+ * Casting to uint64_t since the value is at most 5 bytes, but will appear as a 32-bit literal if it
+ * is less, giving a warning when right-shifting by 32.
+ */
+static uint8_t dis_system_id[8] = {BT_BYTES_LIST_LE40((uint64_t)CONFIG_BT_DIS_SYSTEM_ID_IDENTIFIER),
+				   BT_BYTES_LIST_LE24(CONFIG_BT_DIS_SYSTEM_ID_OUI)};
```

Comment:
I'm a bit confused by the Compliance check doesn't complain about this, but AFAIK There should always be a space after `{` and before `}`.

Reply:
If I add space there, it will be removed by `clang-format`

Reply:
```
❯ git grep '{ '|wc -l
   11259
❯ git grep ' }'|wc -l
   14137
```

Reply:
Conversely:
```
❯ git grep '{[0-9,a-z,A-Z]' -- '*.[ch]'|wc -l
    4524
❯ git grep '[0-9,a-z,A-Z]}' -- '*.[ch]'|wc -l
    5617
```

And the earlier statistic filtered to only contain c-code:
```
❯ git grep '{ ' -- '*.[ch]'|wc -l
   10379
❯ git grep ' }' -- '*.[ch]'|wc -l
    7851
```

Reply:
There doesn't seem to be anything in out `.clang-format` file that explicitly specifies this style detail. Looking at the clang-format documentation, the only option I found which impacts it is `Cpp11BracedListStyle: false`.

Reply:
I tried to set this up for another project, and I could not figure out how its done. I think I tried `Cpp11BracedListStyle`.

Reply:
I don't think our style guide has an entry on this, and since most of the BT team has switched to using clang-format, it's going to be an uphill battle to add spaces before and after curly brackets

Reply:
> I don't think our style guide has an entry on this, and since most of the BT team has switched to using clang-format, it's going to be an uphill battle to add spaces before and after curly brackets

That means that the code will gradually move in a more and more inconsistent direction, until the clang-format style starts taking over (peak inconsistency) and then we gradually move to being consistent with clang-format across the code base. In this case, I think the correct fix is to make sure we move towards the most common existing practice in the code base, i.e. let's fix the clang-format configuration instead.

Reply:
Agreed :) We just need to agree on a style then (`{ 0 }` seems to be preferred over `{0}`).